src/rum/instrumentation/DdRumReactNativeNavigationTracking.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import React from 'react'
+import { Navigation } from 'react-native-navigation';
+import { DdRum } from '../../foundation';
+
+/**
+* Provides RUM auto-instrumentation feature to track user interaction as RUM events.
+* For now we are only covering the "onPress" events.
+*/
+export default class DdRumReactNativeNavigationTracking {
+
+    private static isTracking = false
+    private static trackedComponentIds : Array<any> = [];
+
+    /**
+     * Starts tracking user interactions and sends a RUM Action event every time a new interaction was detected.
+     * Please note that we are only considering as valid - for - tracking only the user interactions that have
+     * a visible output (either an UI state change or a Resource request)
+     */
+    static startTracking(): void {
```

Comment:
shouldn't we provide `stopTracking` counterpart as well?

Reply:
@xgouchet Can you please check it? I guess we need it to be consistent with `DdRumReactNavigationTracking`